backends/dpdk/dpdkArch.h

```diff
@@ -674,6 +674,11 @@ class CollectExternDeclaration : public Inspector {
                     ::error(ErrorType::ERR_EXPECTED,
                             "%1%: expected size and optionally init_val as arguments", d);
                 }
+            } else if (externTypeName == "MatchValueLookupTable") {
```

Comment:
I think CR25 is expecting not to emit error if the const entries and default values are not provided. Please check and update accordingly